### ISSUE TITLE
TKMM doesn't run after install

### ISSUE THREAD

**Issue by RowanSkie** [TKMM-Team/Tkmm#33]
I just installed TKMM, and it doesn't want to run at all. No console or exceptions. Windows 11 64bit. with required .nET downloaded and installed as well.

**Comment by ArchLeaders** [comment]
Does the launcher run?

**Comment by RowanSkie** [comment]
The launcher runs and installs, but both standalone TKMM and the installed TKMM doesn't open.

**Comment by ArchLeaders** [comment]
Open `Command Prompt (cmd)` and type `tkmm` then press enter.

Please send a screenshot or the contents of the window.

**Comment by ArchLeaders** [comment]
Also, just so you know, standalone TKMM doesn't work. The exe included in the release is just for the installer.

**Comment by RowanSkie** [comment]
> Open `Command Prompt (cmd)` and type `tkmm` then press enter.
> 
> Please send a screenshot or the contents of the window.

The thing is, it also closes the console, so I can't take a picture or even force Windows Terminal to override when it closes.

**Comment by ArchLeaders** [comment]
Open `%localappdata%\tkmm\Config.json` and set `ShowConsole` to `true`.

```json
{
    "ShowConsole": true
}
```

After that, save the file and try again. The command prompt window should stay open.

**Comment by RowanSkie** [comment]
> Open `%localappdata%\tkmm\Config.json` and set `ShowConsole` to `true`.
> 
> ```json
> {
>     "ShowConsole": true
> }
> ```
> 
> After that, save the file and try again. The command prompt window should stay open.

It still failed...

**Comment by ArchLeaders** [comment]
That won't fix the problem, it will just make the console stay open so you can read the error.

**Comment by RowanSkie** [comment]
> That won't fix the problem, it will just make the console stay open so you can read the error.

No, I meant it still kept closing the console.

**Comment by ArchLeaders** [comment]
I think you probably did not edit the file correctly. Can you send the edited file contents so I can verify this?

**Comment by RowanSkie** [comment]
The app didn't create a config.json file so I had to make one myself.
[Config.json](https://github.com/user-attachments/files/16647027/Config.json)

**Comment by ArchLeaders** [comment]
Oh interesting, that might be the key to the problem. I'll see if I can recreate it. Thanks.

**Comment by ArchLeaders** [comment]
Seems to work fine for me. Any chance the main exe (`%localappdata%\tkmm\bin\tkmm.exe`) is being blocked by an antivirus?

**Comment by RowanSkie** [comment]
> Seems to work fine for me. Any chance the main exe (`%localappdata%\tkmm\bin\tkmm.exe`) is being blocked by an antivirus?

Not blocked by an antivirus, it's on the exclusion list too.

**Comment by ArchLeaders** [comment]
Very odd. My only advice is to delete the folders `%localappdata%\tkmm`, `%localappdata%\totk`, and `%appdata%\tkmm` to see if that might be causing the issue.

It might also be worth verifying that the .net runtime is installed, although the launcher requires this as well so it should be installed.

You can check that by typing `dotnet --list-runtimes` in CMD.

---

If it's still not working, I'll send a custom build of the exe that will not hide the console so you can check the error log.

**Comment by RowanSkie** [comment]
Ok, now it worked. I just previously deleted the tkmm folders.
